keps/sig-node/1977-container-notifier/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: Container notifier
+kep-number: 1977
+authors:
+  - "@xing-yang"
+  - "@yuxiangqian"
+owning-sig: sig-node
+participating-sigs:
+  - sig-storage
+status: implementable
+creation-date: 2020-09-20
+reviewers:
+  - "@thockin"
+  - "@liggit"
+  - "@saad-ali"
+  - "@sjenning"
+approvers:
```

Comment:
That's a lot of approvers - do we need them all?  I feel like Jordan OR me, Dawn OR Derek..

Reply:
Sure, we'll trim down the approvers list.